src/views/ResourceManagement/SystemParameters/RuntimeProcessorDiagnostic.vue

```diff
@@ -353,7 +355,7 @@ export default {
               )
             );
           } else {
-            this.successToast(message);
+            this.successToast(message[0]);
```

Comment:
Why is this message[0]?

Reply:
So that the number as highlighted in the screenshot doesn't get displayed. 

<img width="1728" alt="Screenshot 2024-10-10 at 11 14 11 AM" src="https://github.com/user-attachments/assets/692f0af8-aabb-4930-a9e6-cfdbcbaf6cfb">

Reply:
Is this change only required in 1110?

Reply:
Yes